common/jdbc/build.gradle

```diff
@@ -9,15 +9,15 @@ dependencies {
     implementation project(':common:rest-jdbc-spring-api')
     implementation group: 'ai.h2o', name: 'mojo2-runtime-api'
     implementation group: 'ai.h2o', name: 'mojo2-runtime-impl'
-    implementation group: 'ai.h2o', name: 'sparkling-water-scoring_2.11'
+    implementation group: 'ai.h2o', name: 'sparkling-water-scoring_2.12'
     implementation group: 'org.scala-lang', name: 'scala-library'
-    implementation group: 'org.apache.spark', name: 'spark-core_2.11'
-    implementation group: 'org.apache.spark', name: 'spark-sql_2.11'
-    implementation group: 'org.apache.spark', name: 'spark-mllib_2.11'
+    implementation group: 'org.apache.spark', name: 'spark-core_2.12'
+    implementation group: 'org.apache.spark', name: 'spark-sql_2.12'
+    implementation group: 'org.apache.spark', name: 'spark-mllib_2.12'
```

Comment:
Hope that versions we have declared at `gradle/mixins/dependencies.gradle` gets applied here correctly.